File: 프로그래머스/unrated/181928. 이어 붙인 수/이어 붙인 수.py
def solution(num_list):
    answer = 0
    odd_str = ''
    even_str = ''
    
    for num in num_list:
        if num % 2 == 0:
            even_str += str(num)
        else:
            odd_str += str(num)
    answer = int(odd_str) + int(even_str)
    
    
    return answer

File: 프로그래머스/0/181928. 이어 붙인 수/이어 붙인 수.java
class Solution {
    public int solution(int[] num_list) {
        int answer = 0;
        String odd_num = "";
        String even_num = "";
        for(int i=0;i<num_list.length;i++){
            if(num_list[i]%2==1){
                odd_num+=Integer.toString(num_list[i]);
            }
            else{
                even_num+=Integer.toString(num_list[i]);
            }
        }
        answer=Integer.parseInt(odd_num)+Integer.parseInt(even_num);
        return answer;
    }
}